src/Pumukit/WebTVBundle/Controller/LegacyController.php

```diff
@@ -110,7 +110,7 @@ public function multimediaObjectAction($pumukit1id)
      * @Route("/{_locale}/mmobj/iframe/id/{pumukit1id}", defaults={"filter": false}, requirements={"_locale"=".."})
      * @Route("/{_locale}/video/iframe/{pumukit1id}.html", defaults={"filter": false}, requirements={"_locale"=".."})
      * @Route("/index.php/{_locale}/video/iframe/{pumukit1id}.html", defaults={"filter": false}, requirements={"_locale"=".."})
-     *
+     * 
```

Comment:
Remove this

Reply:
Closed by https://github.com/pumukit/PuMuKIT/pull/209